src/event.lib.php

```diff
@@ -766,14 +771,16 @@ public static function setNewPassword($user_id = 0)
      * @since 1.8.25
      *
      * @param WP_User $user WP_User object.
-     * @return string|false Returns a password reset key as a string, false otherwise.
+     * @return string|WP_Error Returns a password reset key as a string, WP_Error otherwise.
      */
     public static function GetPasswordResetKey($user)
```

Comment:
It seems this method is called only from within the same class. Do you think its visibility could be changed from `public` to `private`?

Reply:
Done as part of https://github.com/Sucuri/sucuri-wordpress-plugin/pull/100/commits/b523d74c97074c176b60d12d81bdaf0c83c40e17 :)